pkg/start/probe.go

```diff
@@ -91,29 +91,29 @@ func (p *Probe) probeTarget(ctx context.Context) error {
 	p.probeCount += 1
 	stats.Record(ctx, metrics.ProbeCount.M(p.probeCount))
 
-	block, teardown, err := p.generateContent(ctx)
+	// Generate Content on the target
+	cid, teardown, err := p.getContent(ctx)
```

Comment:
`cid` collides with the package name which is unfortunate. I'd rather name it differently. E.g. `contentID` or just `c`.